Resources/Prototypes/Objectives/traitorObjectives.yml

```diff
@@ -191,4 +191,3 @@
   conditions:
     - !type:StealCondition
       prototype: JetpackCaptainFilled
-      owner: job-name-captain
```

Comment:
owner is currently required. It needs to be made optional.

Reply:
:smilethink: